stripe/src/test/java/com/stripe/android/StripeApiRepositoryTest.kt

```diff
@@ -60,9 +58,8 @@ import org.robolectric.RobolectricTestRunner
 @ExperimentalCoroutinesApi
 class StripeApiRepositoryTest {
     private val testDispatcher = TestCoroutineDispatcher()
-    private val testScope = TestCoroutineScope(testDispatcher)
 
-    private val context: Context = ApplicationProvider.getApplicationContext()
+    private val context = ApplicationProvider.getApplicationContext<Context>()
```

Comment:
I actually prefer what you had before since the type is more explicit and more resilient to refactoring.

Reply:
Done!